Initial Website

1337kiwi opened the pull request:
Hi all. First Commit. Revamped the website, added in goodies, and so on. 
Moving TItle is up for debate. I personally like it but I think we should vote on it, 
Support for most browsers (list in comments), links to other resources work, and so on. Complete change of the website structure, linking to the demo site added.

Regarding Text/Colours: I have synesthesia which means I perceive colours in super weird ways and identify colours differently (7 is yellow gang WYA). I also have no clue how to write things in advertiser-ey ways. I've used colours to match the branding colour scheme. Please give as many ideas or recommendations for colour scheme and/or text changes as you can.  Credit to @Bankst for the favicon.

1337kiwi committed:
Initial Website

Hi all. First Commit. Revamped the website, added in goodies, and so on.
Moving TItle, Support for most browsers, links to other resources, and so on. If the moving title isn't welcome, just delete the title.js file and change the title in index.html. Credit to @Bankst for the favicon.

1337kiwi committed:
Favicon Updating + Getting rid of needless images.

Added 16 and 32 bit favicons, made the 32 bit one the standard (I think). Got rid of extra images since the Logos PNG files are now a part of the website.

1337kiwi committed:
[Main Page] Branding Fix

Fixed the naming to match branding.

1337kiwi committed:
[Colours] Matched Branding

Changed the colours to match the official branding colours in https://github.com/PhotonVision/photonvision-branding/blob/master/PhotonVision%20Branding/Documentation/Photon%20Vision%20Branding%20Standards.pdf

1337kiwi committed:
[Colours] Background

Matched body background with main background.

1337kiwi committed:
[Header] Title/Image

Commented out the picture for the header title, instead made it the "The Future is in Sight" tagline. Picture commented out in case someone makes tagline picture in the future, or if the community wants to switch back.

1337kiwi commented:
Tested on Linux Firefox and Chromium. Used User-Agent Switcher to test on these platforms:

Mac OS X / Safari 12

Windows / IE 11
Windows / Edge 44
Windows / Chrome 83
Windows / Firefox 78
Windows / Firefox 60 ESR

Android Phone / Firefox 78
Android Phone / Chrome 83
Android Tablet / Firefox 78
Android Tablet / Chrome 83

iPhone / Safari 12.1.1
iPad / Safari 12.1.1

Google Bot
PS4
Curl
Wget

neel-j-dev commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Project landing page (like Chameleon has)](https://trello.com/c/9KP0vgU6/25-project-landing-page-like-chameleon-has)

1337kiwi committed:
[Header] Left Side temp change

1337kiwi committed:
[Landing] Button Restructure

Made the button locations a bit better, colours easier on the eyes.

1337kiwi committed:
[Header]

Fixed text to make it clearer and simpler.

1337kiwi committed:
[Title] Reverted Title

Reverted the title back to static, fixed GitHub name.

prateekma commented on index.html:
```diff
@@ -1,10 +1,9 @@
 <html> <!-- TODO: We need more images and videos to improve quality of the site, as well as other design stuff-->
     <head>
-        <title>P</title>
+        <title>Photon</title>
```
I think the title should be "Photon Vision" instead of just "Photon".

1337kiwi committed:
[Landing] Fixes and Features

Fixed the arrow, added 2 features to replace the placeholders.

1337kiwi committed:
[Header] Left Header Title

Need to work on this later, unless someone else is willing to.

1337kiwi committed:
[Update]

Recent import from d4rkf0x.github.io. Everything there applied.

1337kiwi committed:
[Downloads] Typo Fix

Typo

1337kiwi committed:
[General] Update to current Github Pages website

Thanks to Modelmat for updating the icons to FontAwesome5, and changing them to different icons based on votes. Man did me a solid by doing so

1337kiwi committed:
[Landing] Color Update

I'm not sure why this didn't save earlier

pietroglyph reviewed (approved):
Looks good enough to merge.
